server/handlers/error.go

```diff
@@ -422,6 +423,9 @@ func ErrSavePattern(err error) error {
 	return errors.New(ErrSavePatternCode, errors.Alert, []string{"Error failed to save pattern"}, []string{err.Error()}, []string{"Cannot save the Pattern due to wrong path or URL"}, []string{"Check if the given path or URL of the Pattern is correct"})
 }
 
+func ErrSaveApplication(err error) error {
```

Comment:
Did you verify that there is the proper error message for unknown components, @Revolyssup?